.github/workflows/terraform_build.yaml

```diff
@@ -41,10 +39,9 @@ jobs:
                               -var "jterry_ip=${JTERRY_IP}" \
                               -var "dmannarino_ip=${DMANNARINO_IP}" \
                               -var "snegusse_ip=${SNEGUSSE_IP}" \
-                              -var "gtempus_ip=${GTEMPUS_IP}" \
-                              -var "office_3sc_ip=${OFFICE_3SC_IP}" \
-                              -var "vpn_3sc_ip=${VPN_3SC_IP}"
+                              -var "gtempus_ip=${GTEMPUS_IP}"
```

Comment:
@dmannarino Just want to confirm that we definitely don't need the 3SC ip's any more?

Reply:
Confirmed. In fact (although I didn't see it in AWS) they said they have their own Bastion in another account. Actually, I hope to clean out all of these soon, as it turns out we can use Session Manager for even port forwarding, and don't need SSH access to Bastion anymore.